src/main/java/com/purbon/kafka/topology/TopologyValidator.java

```diff
@@ -119,9 +124,18 @@ public TopologyValidator(Configuration config) {
 
   private Class<?> getValidationClazz(String validationClass) {
     try {
-      return Class.forName(validationClass);
+      return Class.forName(getValidationClassName(validationClass));
     } catch (ClassNotFoundException e) {
       return null;
     }
   }
+
+  private String getValidationClassName(String validationClass) {
+    return validationClass.split(CLASS_PARAM_DELIMITER)[0];
+  }
+
+  private String getValidationStringParam(String validationClass) {
+    String[] strings = validationClass.split(CLASS_PARAM_DELIMITER);
+    return strings.length > 1 ? strings[1] : null;
```

Comment:
Generally speaking it would be good to use Optional here https://stackoverflow.com/questions/28746482/optional-vs-null-what-is-the-purpose-of-optional-in-java-8